examples/bash_readline.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2016 Kinvolk
```

Comment:
you can use your own name and/or company name if you wrote the code.

Also, s/2016/2017.